### PR TITLE
🚮  Remove RENDLE_ON_IDLE experiment logic

### PR DESCRIPTION
**Pull request by powerivq**
Wait till https://github.com/ampproject/amphtml/pull/29660 rolls out and removes this experiment code for render-on-idle-fix.

### CODE REVIEW

**Comment by powerivq** [comment]
need @lannka  to sign off the owner approval.

**Review by lannka — APPROVED** [review]

**Comment by powerivq** [comment]
@jridgewell for owner approval. (this change was originally from choumx@)

**Review by jridgewell — APPROVED** [review]

File: build-system/global-configs/prod-config.json [hunk 1]
```diff
@@ -36,5 +36,5 @@
   "amp-ad-no-center-css": 0,
   "analytics-chunks": 1,
   "sticky-ad-padding-bottom": 1,
-  "render-on-idle-fix": 1
+  "ad-adsense-gam-round-params": 0.1
```
Comment (jridgewell) [inline build-system/global-configs/prod-config.json:39]: Is this intentional?

**Review by jridgewell — CHANGES_REQUESTED** [review]

File: build-system/global-configs/prod-config.json (hunk 1 above)
Comment (powerivq) [reply build-system/global-configs/prod-config.json]: great catch!

**Review by jridgewell — APPROVED** [review]

**Pull request merged**